Framework/Algorithms/src/CalculateCarpenterSampleCorrection.cpp

```diff
@@ -0,0 +1,467 @@
+#include "MantidAlgorithms/CalculateCarpenterSampleCorrection.h"
+#include "MantidAPI/AlgorithmManager.h"
```

Comment:
Do we also need all of these includes? It would be good for future maintenance to keep this list as small as possible.